src/uct/ib/rdmacm/rdmacm_cm.h

```diff
@@ -25,6 +25,17 @@ KHASH_MAP_INIT_INT64(uct_rdmacm_cm_device_contexts, struct uct_rdmacm_cm_device_
 #define UCT_RDMACM_EP_STRING_LEN                192   /** A string to hold the ep info */
 
 
+/**
+ * Reserved qpn mode.
+ */
+typedef enum uct_rdmacm_cm_reserved_qpn_mode {
+    UCT_RDMACM_CM_RESERVED_QPN_YES,
+    UCT_RDMACM_CM_RESERVED_QPN_TRY,
+    UCT_RDMACM_CM_RESERVED_QPN_NO,
+    UCT_RDMACM_CM_RESERVED_QPN_LAST
+} uct_rdmacm_cm_reserved_qpn_mode_t;
```

Comment:
no need to define it: reuse ucs_ternary_auto_value_t

Reply:
@yosefe We don't need auto, but reuse ucs_ternary_auto_value_t has auto and cause the UCS_TERNARY_LAST not the same the LAST value in our case. Maybe we can:
1） Add another definition similar as ucs_ternary_auto_value_t but without UCS_AUTO.
or
2)  add "auto" config in uct_rdmacm_cm_reserved_qpn, but just don't use it.

Reply:
it should be ok. ucs_config_sscanf_ternary() already expects only yes/no/try. we never really use UCT_RDMACM_CM_RESERVED_QPN_LAST/UCS_TERNARY_LAST  in the code.